pkgs/samples/lib/gemini.dart

```diff
@@ -0,0 +1,336 @@
+// Copyright 2024 the Dart project authors. All rights reserved.
```

Comment:
Should this be a built-in sample or one that we provide via a gist?

Reply:
I'll take it out for now. Here's the gist: https://gist.github.com/flutterdevrelgists/341bc46b2ed1d2055d357ab987ed5fc2